### PR TITLE
Formatting, readability

### PR DESCRIPTION
**Pull request by kirilltitov**
Mostly it's about 120 characters per line.

### CODE REVIEW

File: Sources/Logging/Logging.swift [hunk 1]
```diff
@@ -454,7 +494,10 @@ extension Logger {
     ///
     ///     logger.info("Hello \(world)")
     ///
-    public struct Message: ExpressibleByStringLiteral, Equatable, CustomStringConvertible, ExpressibleByStringInterpolation {
+    public struct Message: ExpressibleByStringLiteral,
+                           Equatable,
+                           CustomStringConvertible,
+                           ExpressibleByStringInterpolation {
```
Comment (kirilltitov) [inline Sources/Logging/Logging.swift]: These lines are questionable. I'm not sure about this formatting convention.

File: Sources/Logging/Logging.swift [hunk 2]
```diff
@@ -97,12 +97,19 @@ extension Logger {
...
     @inlinable
-    public func log(level: Logger.Level,
-                    _ message: @autoclosure () -> Logger.Message,
-                    metadata: @autoclosure () -> Logger.Metadata? = nil,
-                    file: String = #file, function: String = #function, line: UInt = #line) {
+    public func log(
+        level: Logger.Level,
+        _ message: @autoclosure () -> Logger.Message,
+        metadata: @autoclosure () -> Logger.Metadata? = nil,
+        file: String = #file, function: String = #function, line: UInt = #line
```
Comment (kirilltitov) [inline Sources/Logging/Logging.swift]: Maybe `file` and `function` should be `StaticString`?

Comment (weissi) [reply Sources/Logging/Logging.swift]: @kirilltitov 

- see #4 , `String` in Swift 5 leads to slightly better code and one byte less storage that needs to be copied per call.
- the indentation doesn't match Xcode's default, I think we should use the existing indentation

```swift
func foo(bar: Bar,
         buz: Buz)
```

File: Sources/Logging/Logging.swift (hunk 1 above)
Comment (weissi) [reply Sources/Logging/Logging.swift]: I'd be fine with this

File: Sources/Logging/Logging.swift (hunk 2 above)
Comment (kirilltitov) [reply Sources/Logging/Logging.swift]: Oh well, I don't like that C-like arguments formatting at all, but whatever :)

**Comment by kirilltitov** [comment]
@weissi addressed your comments 👍

File: Sources/Logging/Logging.swift [hunk 3]
```diff
@@ -102,7 +102,12 @@ extension Logger {
                     metadata: @autoclosure () -> Logger.Metadata? = nil,
                     file: String = #file, function: String = #function, line: UInt = #line) {
         if self.logLevel >= level {
-            self.handler.log(level: level, message: message(), metadata: metadata(), file: file, function: function, line: line)
+            self.handler.log(
+                level: level,
```
Comment (weissi) [inline Sources/Logging/Logging.swift]: @kirilltitov mind fixing the indentation here too?

Comment (kirilltitov) [reply Sources/Logging/Logging.swift]: Oof, right. Done.

**Pull request merged**